### PR TITLE
Improve external usability

### PR DESCRIPTION
**Pull request by Adafede**
Hi @baoilleach!

Thank you for this great implementation!

Here below a few suggestions that would probably help external users (issues I faced while trying to use it out of the box)

### CODE REVIEW

File: smizip/scripts/find_best_ngrams.py [hunk 1]
```diff
@@ -191,38 +193,56 @@ def main():
...
             elif len(ngram) > 1:
                 multichars.append(ngram)
-    out.write(f"The initial list of single-char ngrams is:\n  {repr(''.join(sorted(singlechars)))}\n")
-    out.write(f"The initial list of multi-char ngrams is:\n  {multichars}\n")
+    out.write(f"The initial list of single-char n-grams is:\n  {repr(''.join(sorted(singlechars)))}\n")
+    out.write(f"The initial list of multi-char n-grams is:\n  {multichars}\n")
 
     orig_num_ngrams = len(singlechars)
     smiles_iter = open(args.input)
+
```
Comment (Adafede) [inline smizip/scripts/find_best_ngrams.py:201]: I had multiple time the issue that the smiles iterator was exhausted before reaching the end else

File: smizip/scripts/find_best_ngrams.py [hunk 2]
```diff
@@ -88,66 +87,69 @@ def __init__(self):
...
             val, is_measured = self.values[ngram]
-            tmp.append( (ngram, val, is_measured, val*count) )
-        tmp.sort(reverse=True, key=lambda x:x[3])
+            tmp.append((ngram, val, is_measured, val * count))
+        tmp.sort(reverse=True, key=lambda x: x[3])
         for t in tmp:
             yield t
 
+# This default list should be more explicit.
+# It took me some time to understand the error "u" because I had compounds with copper (Cu).
```
Comment (Adafede) [inline smizip/scripts/find_best_ngrams.py:139]: Probably also add a small note to the README about this?

**Comment by baoilleach** [comment]
Hi @Adafede, thanks for your comments and code. The commits here cover a whole lot of different things, which make it difficult to review, e.g. whitespace changes, changes to the definition of fast/slow. Can you do separate PRs? Also, you may wish to discuss the PRs as issues first, especially if proposing major changes.

**Comment by Adafede** [comment]
Good! Will close this one and re-open other ones.

**Pull request closed without merging**